src/braket/tasks/local_quantum_task.py

```diff
@@ -39,13 +39,23 @@ def __init__(
 
     @property
     def id(self) -> str:
+        """Gets the task ID.
+
+        Returns:
+            str: The ID of the task.
+        """
         return str(self._id)
 
     def cancel(self) -> None:
         """Cancel the quantum task."""
```

Comment:
Would this need to have a Raises NotImplementedError section since it raises an error?

Reply:
For single line not implemented functions, it is accepted to omit the raises exception.